src/docs/asciidoc/cdc-configuration.adoc

```diff
@@ -396,7 +396,7 @@ If polling fails, reader will try again using the specified interval.
 | Pipeline property name
 
 | eventuate.database.schema
-| Schema which is listened by the CDC service
+| Schema which is listened by the CDC service (use `NONE` for default from jdbc connection)
```

Comment:
I don't think `eventuate.database.schema` is  used to configure the CDC. 
It's used to configure the application/service.

Reply:
It is used. Each reader has datasource. But I added info for application.